.github/workflows/pr.yml

```
@@ -75,9 +99,9 @@ jobs:
     -
       name: Check deployment files
       run: |
-        make generate_default_deployment
+        make generate_default_deployment generate_olm_bundle_yaml
         if [[ ! -z $(git status -s) ]]; then
-          echo "Generated deployment files are out of date; run make generate_default_deployment."
+          echo "Generated deployment files are out of date; run `make generate_default_deployment generate_olm_bundle_yaml`."
```

I think this will re-evaluate `make generate_default_deployment generate_olm_bundle_yaml` -- e.g.
```
$ echo "Hi `echo hello`"
Hi hello
```
A workaround would be to use single quotes instead,
```suggestion
          echo 'Generated deployment files are out of date; run `make generate_default_deployment generate_olm_bundle_yaml`.'
```
but that feels error-prone.

---

I'll take a look but the blocker for this PR is that operator-sdk behaves differently locally and on github, despite of fact (as far as I checked) the same version is used:

> time="2021-06-08T11:51:54Z" level=fatal msg="invalid command options: none of --input-dir, --deploy-dir, or --crds-dir may be set if reading from stdin"
make: *** [build/make/olm.mk:16: generate_olm_bundle_yaml] Error 1

https://github.com/devfile/devworkspace-operator/pull/449/checks?check_run_id=2773442065

---

applied suggestion